src/main/java/org/opensearch/sdk/handlers/GenericResponseHandler.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.sdk.handlers;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.extensions.ExtensionBooleanResponse;
+import org.opensearch.threadpool.ThreadPool;
+import org.opensearch.transport.TransportException;
+import org.opensearch.transport.TransportResponseHandler;
+
+import java.io.IOException;
+
+/**
+ * This class handles the response {{@link org.opensearch.extensions.ExtensionBooleanResponse }} from OpenSearch to Extension.
+ */
+public class GenericResponseHandler implements TransportResponseHandler<ExtensionBooleanResponse> {
```

Comment:
Can we name this class as `ExtensionResponseHandler`?

Reply:
Also, can we use this handler for other response handler we have as well?

Reply:
Sure. I like `ExtensionResponseHandler`.